doc/user/pathd.rst

```diff
@@ -471,7 +471,7 @@ Configuration Commands
    Disable or start the definition of a PCC.
 
 
-.. clicmd:: msd (1-32)
+.. clicmd:: msd [(1-32)]
```

Comment:
For me it's unclear when reading here. Seems I'm able to enter just "msd", and what is the effect of that? Can it be clarified?

Reply:
which behaviour do you expect?

Reply:
No idea what would be the behavior here, is this something that is picked by default if I do not specify the depth?

Reply:
In general, there is a default value for MSD which depend of the hardware. For Linux Kernel, the default value is 32. This CLI is used to specify a different value as the default one. Thus, the CLI should keep the (1-32) syntax to force specifying the new value of MSD. The `no MSD` CLI acts as go back to the default value.
So, it is preferable to not change this.

Reply:
@odd22 ,@ton31337, please review